src/Common/src/Interop/Linux/libc/Interop.inotify.cs

```diff
@@ -32,7 +32,7 @@ internal static int inotify_rm_watch(int fd, int wd)
                 }
                 else
                 {
-                    System.Diagnostics.Debug.Fail("inotify_rm_watch failed with " + hr);
+                    global::System.Diagnostics.Debug.Fail("inotify_rm_watch failed with " + hr);
```

Comment:
Sure. I can use some help coming up with the right names.

Each shim library has three names associated with it.

1. The file name: e.g. System.Security.Cryptography.Native.(so|dylib)
2. The constant in Interop.Libraries: e.g. Interop.Libraries.CryptoInterop
3. The partial class inside Interop that exposes P/Invokes: Currently, Interop.NativeCrypto

Some things that are bothering about status quo that I was trying to address here:

* We have to type (3) at every call site, so shorter is better with all else equal and Native at the end is somewhat redundant with Interop at the beginning.

* (2) can be longer with less pain, and since it directly represents the file name, I think a 1:1 convention between constant and file name is better.

So I was trending towards:

* System.Native.so, Interop.Libraries.SystemNative, Interop.System.SomeExport
* System.Security.Cryptography.so, Interop.Libraries.SystemSecurityCryptographyNative, Interop.Cryptography.SomeExport

(We could also go with short names for the *.so files and then the existing conventions for e.g. libc work fine, but it's hard to come up with something systematic that works well with the long names. A lot of folks seemed to like the long names, so I'm not sure I want to revisit it again...)